.claude/ref/CosmWasmBook/Basics/Building the contract - CosmWasm book.md

```
@@ -0,0 +1,45 @@
+created: 2025-11-05
+description: "Guide to building CosmWasm smart contracts"
+tags:
+  - "clippings"
+status: "unread"
+---
+いよいよコントラクトのビルド作業に入ります。ローカル環境でのテスト用には、従来のcargoビルドパイプラインを使用できます： `cargo build` でコンパイルを行い、 `cargo test` ですべてのテストを実行します（現時点ではテストケースが存在しませんが、近日中に実装する予定です）。
+
+ただし、ブロックチェーンにコントラクトをデプロイするためには、wasmバイナリを作成する必要があります。ビルドコマンドに追加の引数を指定することでこれを実行できます：
+
+```js
+$ cargo build --target wasm32-unknown-unknown --release
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)